### PR TITLE
Miscellaneous

### PR DESCRIPTION
**Pull request by OnkelUlla**
Again some small patches in the context of RAUC's destroy functions.

### CODE REVIEW

File: src/bundle.c [hunk 1]
```diff
@@ -552,9 +552,10 @@ gboolean umount_bundle(RaucBundle *bundle, GError **error) {
 }
 
 void free_bundle(RaucBundle *bundle) {
-
-	g_free(bundle->path);
-	if (bundle->verified_chain)
-		sk_X509_pop_free(bundle->verified_chain, X509_free);
-	g_free(bundle);
+	if (G_LIKELY(bundle)) {
```
Comment (jluebbe) [inline src/bundle.c]: Can we use the inverse? if (!bundle) return? That would avoid indenting the whole function.

Comment (ejoerns) [reply src/bundle.c]: The glib functions use `g_return_if_fail()` in these cases (to check for passing NULL pointers). The downside of this is that we would have to use `g_clear_pointer()` for calling the free function on potential NULL pointers, but this is what we have to do for glib free functions (Except `g_free()`) anyway.

Comment (OnkelUlla) [reply src/bundle.c]: I changed it to `g_return_if_fail()` as requested.
Digging a little bit more through GLib, reveals that this scheme is not used consitently, though. See [gchecksum](https://github.com/GNOME/glib/blob/master/glib/gchecksum.c#L1552) for example.

**Comment by OnkelUlla** [comment]
Did the change to `g_return_if_fail()` as requested and rebased to current master.

**Comment by ejoerns** [comment]
Travis is now fixed by #167. Please rebase and push again.

**Comment by OnkelUlla** [comment]
Rebased and pushed again. :-)

**Comment by OnkelUlla** [comment]
Rebased onto Enrico's coveralls fix.

**Pull request merged**